src/main/java/uk/gov/hmcts/reform/judicialapi/repository/UserProfileRepository.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.judicialapi.repository;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.reform.judicialapi.domain.UserProfile;
+
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public interface UserProfileRepository extends JpaRepository<UserProfile, UUID> {
+
+    Page<UserProfile> findBySidamIdIn(List<String> sidamIds, Pageable pageable);
```

Comment:
I think it should be extends JpaRepository<UserProfile, String> 

UUID would increase cost of operations I feel.